addons/binding/org.openhab.binding.amazonechocontrol/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,425 @@
+            <channel id="amazonMusicTrackId" typeId="amazonMusicTrackId" />
+            <channel id="amazonMusicPlayListId" typeId="amazonMusicPlayListId" />
+            <channel id="amazonMusicPlayListIdLastUsed" typeId="amazonMusicPlayListIdLastUsed" />
+            <channel id="amazonMusic" typeId="amazonMusic" />
+            <channel id="remind" typeId="remind" />
+            <channel id="playAlarmSound" typeId="playAlarmSound" />
+            <channel id="playFlashBriefing" typeId="playFlashBriefing" />
+            <channel id="playTrafficNews" typeId="playTrafficNews" />
+            <channel id="playWeatherReport" typeId="playWeatherReport" />
+            <channel id="playGoodMorning" typeId="playGoodMorning" />
+            <channel id="startRoutine" typeId="startRoutine" />
+            <channel id="playMusicProvider" typeId="playMusicProvider" />
+            <channel id="playMusicVoiceCommand" typeId="playMusicVoiceCommand" />
+        </channels>
+        <representation-property>serialNumber</representation-property>
+        <config-description>
+            <parameter name="serialNumber" type="text" required="true">
+                <label>Serial Number</label>
+                <description>You will find the serial number of your device in the Alexa app</description>
+            </parameter>
+        </config-description>
+    </thing-type>
+    <thing-type id="wha">
+        <supported-bridge-type-refs>
+            <bridge-type-ref id="account" />
+        </supported-bridge-type-refs>
+        <label>Amazon Echo Whole House Audio Control</label>
+        <description>Amazon Multiroom Music</description>
+        <channels>
+            <channel id="player" typeId="player" />
+            <channel id="shuffle" typeId="shuffle" />
+            <channel id="imageUrl" typeId="imageUrl" />
+            <channel id="title" typeId="title" />
+            <channel id="subtitle1" typeId="subtitle1" />
+            <channel id="subtitle2" typeId="subtitle2" />
+            <channel id="providerDisplayName" typeId="providerDisplayName" />
+            <channel id="radioStationId" typeId="radioStationId" />
+            <channel id="radio" typeId="radio" />
+            <channel id="amazonMusicTrackId" typeId="amazonMusicTrackId" />
+            <channel id="amazonMusicPlayListId" typeId="amazonMusicPlayListId" />
+            <channel id="amazonMusicPlayListIdLastUsed" typeId="amazonMusicPlayListIdLastUsed" />
+            <channel id="amazonMusic" typeId="amazonMusic" />
+        </channels>
+        <representation-property>serialNumber</representation-property>
+        <config-description>
+            <parameter name="serialNumber" type="text" required="true">
+                <label>Serial Number</label>
+                <description>You will find the serial number of your device in the Alexa app</description>
+            </parameter>
+        </config-description>
+    </thing-type>
+    <thing-type id="unknown">

```

Comment:
No, I don't think so, because it will also be needed for manual creation. Currently all the devices have the same channels, except the Whole House Audio which does not support all of them. But I except different channels in the future, for e.g. maybe the Echo Show can get channel for displaying an image. The unkown device is for users which have a echo device, which does not exit in the moment of releasing the binding. The special feature of the unknown device is, that ALL channels are defined, so the user can use the channels which are working for the new unknown device.